Bio/Blast/ParseBlastTable.py

```diff
@@ -14,6 +14,8 @@
 
 
 class BlastTableEntry(object):
+    """Store the record details."""
```

Comment:
Maybe something about Blast Table Entry, since "record" will have more than one interpretation in the context?

Reply:
I think is better worded now.

Reply:
Better, yes.

Reply:
Bow's suggestion is even better (see separate comments), ``Container for Blast table entry data.``